### PR TITLE
Add note about allowed characters for devices/tags

### PR DESCRIPTION
**Pull request by kprosise**
Added notes in troubleshooting/FAQ and tutorial section on tags.

Checked rendered html, no issues spotted.

This commit addresses FFTK-1897

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Adds frequently requested information.

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

We can add/move the info elsewhere or expand on it, if need be.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e864c59df58770c4b98a84f61dff564035afa981 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2127/docs/artifacts/html/index.html

File: source/tutorials/working-with-tags/following-specific-tag.rst [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+Following a Specific Tag
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 Before configuring a device to follow a specific tag, you must have at least one 
 **Target** tagged with the new tag.
 
 Let's start tagging all **Targets** in the latest version with the tag ``devel`` and ``tutorial``.
 
+.. tip::
+   For tags, you may use the following characters: a-z, A-Z, 0-9, underscore, and dash.
```
Comment (kprosise) [inline source/tutorials/working-with-tags/following-specific-tag.rst:10]:
```suggestion
   For tags, you may use the following characters: a–z, A–Z, 0–9, underscore, and dash. 
```

File: source/user-guide/troubleshooting/troubleshooting.rst [hunk 2]
```diff
@@ -440,6 +440,11 @@ Follow these steps to do so:
 Tips and Abouts
 ---------------
 
+Allowed Characters for Device Names and Tags
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Along with [a–z], [A–Z], and [0–9], underscores and dashes may be used for device names and tags.
```
Comment (vkhoroz) [inline source/user-guide/troubleshooting/troubleshooting.rst:446]: @kprosise please refer to these rules for device names and tags: https://git.foundries.io/development/cloudplatforms/infrastructure/ota-lite/-/blob/master/ota_lite/api/_tuf_validators.py#L10

Basically, for device names & uuids we support these characters: `a-zA-Z0-9_-.`, while for tags we also support a `+` in addition to that list. IIRC a support for a plus sign was specifically requested by one of our customers as that is a usual symbol for annotated git tags.

Note: a validation for build tags and device tags is essentially the same, as that is the same entity for us (like a foreign key reference in database terms).

Comment (kprosise) [reply source/user-guide/troubleshooting/troubleshooting.rst:446]: @vkhoroz Thanks for the review, and explaining it better! I appreciate it!

**Review by angolini — APPROVED** [review]
other than the comments, LGTM

**Comment by doanac** [comment]
Docs for d41b62b42438c44d36c61b5ea128a1c3944ab395 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2135/docs/artifacts/html/index.html

**Review by vkhoroz — APPROVED** [review]
LGTM

**Pull request merged**